sqld/src/query.rs

```diff
@@ -389,6 +391,19 @@ mod test {
         assert_eq!(stmt.expanded_sql().unwrap(), "SELECT 10 || 20");
     }
 
+    #[test]
+    fn test_bind_params_positional_named_no_prefix() {
+        let con = rusqlite::Connection::open_in_memory().unwrap();
+        let mut stmt = con.prepare("SELECT :first || $second").unwrap();
```

Comment:
What happens if you try with a statement like 'select :first || $first'?

Reply:
It won't allow it:
```
missing parameters, expected 2 found 1
```
here's the change:
```
diff --git a/sqld/src/query.rs b/sqld/src/query.rs
index b75bdf5..739f689 100644
--- a/sqld/src/query.rs
+++ b/sqld/src/query.rs
@@ -394,10 +394,9 @@ mod test {
     #[test]
     fn test_bind_params_positional_named_no_prefix() {
         let con = rusqlite::Connection::open_in_memory().unwrap();
-        let mut stmt = con.prepare("SELECT :first || $second").unwrap();
+        let mut stmt = con.prepare("SELECT :first || $first").unwrap();
         let mut params = HashMap::new();
         params.insert("first".to_owned(), Value::Integer(10));
-        params.insert("second".to_owned(), Value::Integer(20));
         let params = Params::new_named(params);
         params.bind(&mut stmt).unwrap();
```

Reply:
When I worked around it by adding second parameter `$first`, it failed:
```
```
here's the change:
```
```

Reply:
Thanks for pointing it out @MarinPostma

Reply:
done.